src/main/external-resources/nsis/setup.nsi

```diff
@@ -372,37 +377,44 @@ Section "Uninstall"
 	Delete /REBOOTOK "$INSTDIR\renderers\Microsoft-Xbox360.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Microsoft-XboxOne.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Miracast-M806.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\Netgear-NeoTV.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Naim-Mu-So.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\etgear-NeoTV.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Netgem-N7700.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Nokia-N900.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\Onkyo-TXNR717.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\OPPO-BDP83.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\OPPO-BDP.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\OPPO-BDP83.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Onkyo-TXNR717.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-DMPBDT.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-DMPBDT220.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-DMPBDT360.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-DMRBWT740.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-SCBTT.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-Viera.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraAS600E.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraAS650.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraCX680.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraCX700.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraE6.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraET60.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraGT50.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraS60.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraST60.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraTHPU30Z.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraTXL32V10E.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Panasonic-VieraVT60.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Philips-AureaAndNetTV.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Philips-PFL.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Philips-Streamium.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Pioneer-BDP.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Pioneer-Kuro.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\PopcornHour.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\README.txt"
```

Comment:
Because i copy paste all the renderer only and forgotten that it had be deleted...